app/controllers/cryptids_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CryptidsController < ApplicationController
   def show
     @cryptid = Cryptid.find(params[:id])
+    @current_user = current_user
```

Comment:
I'm fairly certain you can access `current_user` in your views without defining this instance variable - did you try and have it not work, or can you update now if not?